dist/stat_data.py

```diff
@@ -333,6 +333,7 @@ def __init__(self, name, desc, flags=''):
     # Dhandle statistics
     ##########################################
     DhandleStat('dh_conn_handle_count', 'connection data handles currently active', 'no_clear,no_scale'),
+    DhandleStat('dh_conn_handle_size', 'connection data handle size'),
```

Comment:
I'd have expected this to have the `no_clear` and `no_scale` options set, otherwise the value will be scaled by t2, and will be reset to 0 when statistics are periodically reset, and never retr

Reply:
It should also have the `size` flag set, because the value is a byte count.